### PR TITLE
OEUI-173: Create Components For Lab Order History

### PR DESCRIPTION
**Pull request by Efosaok**
#### JIRA TICKET NAME
[Create Components For Lab Order History](https://issues.openmrs.org/browse/OEUI-173)

#### Summary
- Build out Past and Active Lab order History section with React

### CODE REVIEW

**Comment by hadijahkyampeire** [comment]
I don't you are supposed to put screenshots on the PR, please refer to this https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by Efosaok** [comment]
thanks, @hadijahkyampeire

**Comment by IAMOTZ** [comment]
Hey @Efosaok!
Nice Job with the completion of this task.
But the Past Lab Orders table seems to be very close to the bottom of its container:
screen shot 2018-07-16 at 3 05 18 pm
Can you add a little bit of padding/margin to make it become something like this:
screen shot 2018-07-16 at 3 07 29 pm

File: app/js/components/labOrderEntry/LabEntryForm.jsx [hunk 1]
```diff
@@ -2,7 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import LabPanelFieldSet from './LabPanelFieldSet';
 import LabTestFieldSet from './LabTestFieldSet';
+import ActiveOrders from './ActiveOrders';
+import PastOrders from './PastOrders';
+import Accordion from '../accordion';
```
Comment (dkayiwa) [inline app/js/components/labOrderEntry/LabEntryForm.jsx]: Is accordion supposed to be small a or capital A?

Comment (Efosaok) [reply app/js/components/labOrderEntry/LabEntryForm.jsx]: The accordion there is referring to a folder and not a file, the code is exported from an index.js file inside the accordion directory.

**Comment by Efosaok** [comment]
I have addressed the coverage issue @topseySuave

**Comment by Efosaok** [comment]
Thanks for your observation, @IAMOTZ, I have addressed that

**Pull request merged**